docker/openchallenges/services/mcp-server-standalone.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  openchallenges-mcp-server:
+    image: ghcr.io/sage-bionetworks/openchallenges-mcp-server:latest
+    container_name: openchallenges-mcp-server-standalone
+    restart: always
+    ports:
+      - '8887:8887'
```

Comment:
We should also use environment variable to set the port of the server (`SERVER_PORT=8887`) too. This will provide the user with a full example on how to set the port to whatever they want.